### PR TITLE
Update select input to include dropdown icon

### PR DESCRIPTION
**Pull request by s-o-h**
Hi @hmarcks - this PR includes updates to the select element markup and adds a utility class to disable the pointer over the added icon.

### CODE REVIEW

File: admin/edit-user.html [hunk 1]
```diff
@@ -69,17 +69,22 @@ <h1 class="mb4 f4 fw6">Add User</h1>
...
-          class="w-100 pa3 br2 bw1 b--black-10 input-reset bg-white black-60"
-        >
-          <option value="">
-            -- Please select a group --
-          </option>
-          <option value="option1">Option 1</option>
-          <option value="option2">Option 2</option>
-          <option value="option3">Option 3</option>
-        </select>
+        <div class="relative w-100">
```
Comment (hmarcks) [inline admin/edit-user.html]: Looks like the w-100 isn't needed

File: admin/edit-user.html [hunk 2]
```diff
@@ -69,17 +69,22 @@ <h1 class="mb4 f4 fw6">Add User</h1>
...
+          <select
+            id="group"
+            class="db w-100 pa3 br2 bw1 b--black-10 input-reset bg-white black-60"
```
Comment (hmarcks) [inline admin/edit-user.html]: I think the db can be removed here since it's inside its own div

**Review by hmarcks — CHANGES_REQUESTED** [review]
Just a couple small things to check out, thanks!

File: admin/edit-user.html (hunk 1 above)
Comment (s-o-h) [reply admin/edit-user.html]: .w-100 removed

File: admin/edit-user.html (hunk 2 above)
Comment (s-o-h) [reply admin/edit-user.html]: .db removed

**Comment by s-o-h** [comment]
Thanks for the review @hmarcks! Just pushed an update

**Pull request merged**